### DISCUSSION TITLE
Backup Rustdesk settings and keys before OS reinstall

### DISCUSSION THREAD

**Discussion by amirrezanet** [rustdesk/rustdesk#7879] [Q&A] [answered]
How can I backup Rustdesk before reinstalling the operating system, so that I can retain the same key for users on the new server without them needing to change it?

**Comment by cleudsoncosta** [comment]
I would also like to know how to make a backup, I have a server and I have several clients, if there is a problem with the server and I have to install a new one, which files should I make a backup of so as not to lose the client connection and have to redo everything all over again?

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk-server-pro/discussions/184
